pylint/checkers/classes.py

```diff
@@ -133,6 +133,24 @@ def _positional_parameters(method):
         positional = positional[1:]
     return positional
 
+def _has_different_parameters_default_value(original, overridden):
+    """
+    Return True if one of the overridden arguments has a default
+    value different from the default value of the original argument
+    """
+    orig_name = [param.name for param in original.args]
+    for param_name in orig_name:
+        try:
+            orig_default = original.default_value(param_name).value
+        except astroid.exceptions.NoDefault:
+            orig_default = None
+        try:
+            over_default = overridden.default_value(param_name).value
+            if orig_default != over_default:
```

Comment:
What happens when default value is `None`?
Is `overridden.default_value(param_name).value` equal to `None` or equal to `nodes.Const(None)`? Maybe we can test for that? (default value equal to None is very common).